db/migrate/20210128021147_add_allow_unknown_sender_topic_replies_to_group.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddAllowUnknownSenderTopicRepliesToGroup < ActiveRecord::Migration[6.0]
+  def up
+    add_column :groups, :allow_unknown_sender_topic_replies, :boolean, default: false
+    DB.exec("UPDATE groups SET allow_unknown_sender_topic_replies = false")
+  end
+
+  def down
+    remove_column :groups, :allow_unknown_sender_topic_replies if column_exists?(:groups, :allow_unknown_sender_topic_replies)
```

Comment:
The `if column_exists?` shouldn't be required here. In what case would the column not exist if the migration has run?